### DISCUSSION TITLE
I have problem with installing whisky with terminal

### DISCUSSION THREAD

**Discussion by kalvinov** [Whisky-App/Whisky#739] [Q&A] [answered]
The problem is on the screenshot. What does it mean? Im not a coder. My mac: mac book pro m2 16 gb 512 ssd.
<img width="570" alt="Снимок экрана 2023-12-28 в 19 51 34" src="https://github.com/Whisky-App/Whisky/assets/155090312/d9c1771f-5573-4247-962b-dc048f6df6e0">

**Comment by IsaacMarovitz** [comment, marked as answer]
You're installing using Intel homebrew (ie running under Rosetta). You need to reinstall ARM brew.
